modules/representation_management/spec/requests/representation_management/v0/pdf_generator_2122a_swagger_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+require Rails.root.join('spec', 'rswag_override.rb').to_s
+require_relative '../../../support/swagger_shared_components/v0'
+
+RSpec.describe 'PDF Generator 21-22a', openapi_spec: 'modules/representation_management/app/swagger/v0/swagger.json',
+                                       type: :request do
+  before do
+    create(:accredited_individual, :with_phone,
+           id: SwaggerSharedComponents::V0.representative[:id])
+  end
+
+  path '/representation_management/v0/pdf_generator2122a' do
+    post('Generate a PDF for form 21-22a') do
+      tags 'PDF Generation'
+      consumes 'application/json'
+      produces 'application/pdf'
```

Comment:
This is true for the 200 response. Errors will produce json. I think the 2122 docs also have this error.

Reply:
The 422 now specifies that it has a json response.